src/System.Collections.NonGeneric/tests/CaseInsentiveComparerTests.cs

```diff
@@ -130,41 +131,40 @@ public static void Ctor_CultureInfo_NullCulture_ThrowsArgumentNullException()
         [InlineData(null, null, 0)]
         public static void DefaultInvariant_Compare(object a, object b, int expected)
         {
-            var cultureNames = new string[]
+            RemoteInvoke(() =>
```

Comment:
If a test just uses CultureInfo.Current{UI}Culture (and not DefaultThreadCurrentCulture), uses a try/finally to reset the culture back to the original, and doesn't await in the middle of that, do we need to RemoteInvoke?